.github/workflows/deploy.yml

```diff
@@ -73,9 +73,9 @@ jobs:
         with:
           python-version: 3.10.1
       - name: install dependencies
-        run: python -m pip install --upgrade pip && python -m pip install -r requirements.txt
+        run: python -m pip install --upgrade pip && python -m pip install bandit safety
       - name: Bandit
-        run: bandit -r ./auth ./demo
+        run: bandit -r . -x config/*,tests/*
```

Comment:
What's the update doing here?

Reply:
we're getting bandit to run on all files excluding (-x) config and test folders 👍